### PR TITLE
Fix integrity when migrating from 9.0 to 9.1

### PR DESCRIPTION
**Pull request by VicDeo**
Backport of https://github.com/owncloud/updater/pull/416
Reproduction steps:
1. Install 9.0.4
2. Update to 9.0.8 using stable channel 
3. Update to 9.1.4 using daily channel 
4. Do an integrity check:
```
> php occ integr:check-core
  - EXTRA_FILE:
    - updater/.gitignore:
      - expected: 
      - current: 7c017a81ca97f082b0d97869336cd41ad0d02e67de4865e33ca031775dc9ea5e9e7f024a4fa8016e58f29e7c4fd90d1633944ffbab37ee4c4ff739443c8d059c
```

to test:
apply the fix on top of 9.0.8 before updating to 9.1.4

### CODE REVIEW

File: src/Utils/Locator.php [hunk 1]
```diff
@@ -102,6 +102,7 @@ public function getUpdaterContent(){
 			'README.md',
 			'.travis.yml',
 			'.scrutinizer.yml',
+			'.gitignore'
```
Comment (PVince81) [inline src/Utils/Locator.php]: add nbproject here too just in case ?

**Comment by PVince81** [comment]
:+1: otherwise

**Comment by VicDeo** [comment]
@PVince81  makes sense. Added

**Comment by davitol** [comment]
> Install 9.0.4
> Update to 9.0.8 using stable channel

kindly remember for updates from 9.0.4.:
https://central.owncloud.org/t/patches-for-known-update-issues-from-version-9-0-4-9-0-5-9-1-0-9-1-1/3549

**Comment by davitol** [comment]
Tested and works. Thanks @VicDeo

**Pull request merged**